packages/wonder-blocks-data/src/util/types.js

```diff
@@ -112,3 +112,40 @@ export type ErrorOptions = {|
      */
     cause?: ?Error,
 |};
+
+export interface ScopedCache {
```

Comment:
I was confused when I first saw this because I we were redefining the interface, but what I had seen before was just the docs for the interface.  It's too bad our doc tools don't provide ways to import type definitions from code into the docs.  No action req'd.